### PR TITLE
Feature/ong view patch

### PR DESCRIPTION
**Pull request by pronguto**

### CODE REVIEW

File: sos_brazil/settings.py [hunk 1]
```diff
@@ -50,6 +50,12 @@
 THIRD_PARTY_APPS = ["rest_framework", "rest_framework.authtoken"]
 
 INSTALLED_APPS = DJANGO_APPS + MY_APPS + THIRD_PARTY_APPS
+# project_base/settings.py
```
Comment (marcelosj3) [inline sos_brazil/settings.py:53]: delete comment

**Review by marcelosj3 — CHANGES_REQUESTED** [review]

File: ongs/permissions.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+from ongs.models import Ong
+
+
+class isOngOwner(BasePermission):
+    def has_object_permission(self, request: Request, _, obj: Ong):
+        owner_method = {"PATCH", "DELETE"}
+        for request.method in owner_method:
+            return True
+        for owner in obj.admins:
```
[the remark this answers is not in the document]
Comment (pronguto) [reply ongs/permissions.py]: Done, not like this but it works.

File: ongs/permissions.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
+# def has_object_permission
```
[the remark this answers is not in the document]
Comment (pronguto) [reply ongs/permissions.py]: ok

File: sos_brazil/settings.py (hunk 1 above)
Comment (pronguto) [reply sos_brazil/settings.py]: ok

File: ongs/serializers.py [hunk 4]
```diff
@@ -29,3 +44,23 @@ def update(self, instance: Ong, validated_data: dict):
...
+    class Meta:
+        model = Ong
+        fields = ["admins"]
+
+    def update(self, instance: Ong, _: dict):
+        admins = self.context["request"].data
+        # print()
+        # print(self.context["request"].user.__dict__["user_id"] in instance.admins.values())
+        # # print(instance.admins.values()[0]["user_id"])
+        # print()
```
Comment (marcelosj3) [inline ongs/serializers.py:61]: delete comment

**Review by gabrielsuch — APPROVED** [review]

File: ongs/permissions.py [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+    def has_permission(self, request, view):
+        admin_methods = {"POST", "PATCH", "DELETE"}
+        if request.method in admin_methods:
+            return request.user.is_authenticated
+        return True
+
+    def has_object_permission(self, request: Request, view, obj: Ong):
+        owner_method = {"PATCH", "DELETE"}
+        for request.method in owner_method:
+            if request.user.__dict__["user_id"] in obj.admins.values():
```
Comment (marcelosj3) [inline ongs/permissions.py:17]: just return like this `return request.user.__dict__["user_id"] in obj.admins.values()`

File: ongs/serializers.py (hunk 4 above)
Comment (pronguto) [reply ongs/serializers.py]: Ok

File: ongs/permissions.py (hunk 5 above)
Comment (pronguto) [reply ongs/permissions.py]: Done!

**Review by marcelosj3 — APPROVED** [review]

**Pull request merged**